app/src/main/java/com/codesoom/assignment/models/TaskManager.java

```diff
@@ -0,0 +1,107 @@
+package com.codesoom.assignment.models;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Task 의 관리를 위해 만들었습니다.
+ */
+public class TaskManager {
+    private static long index = 0;
+    private static final HashMap<Long, Task> tasks = new HashMap<>();
+
+    /**
+     * findAll 의 역할을 합니다.
+     * @return 전체 Task 를 List 로 반환합니다.
+     */
+    public static List<Task> find() {
+        return new ArrayList<>(tasks.values());
+    }
+
+    /**
+     * findOne 의 역할을 합니다.
+     * @param id 찾고자 하는 Task id
+     * @return 존재 시 Task 반환, 없으면 null 반환
+     */
+    public static Task find(long id) {
+        return tasks.get(id);
+    }
+
+    /**
+     * Task 를 받아서 insert 합니다. id가 존재시 Exception 을 반환합니다.
```

Comment:
`@throws`가 있으므로 생략해도 좋습니다.
```suggestion
     * Task 를 받아서 insert 합니다.
```